kylecornelissen/whats-cookin
# Search functionality

#### What's this PR do?
- Added case-insensitive functionality for search bar to search based off of the user input value
- Fixed the aside side bar to fit whatever size the page is
#### Where should the reviewer start?
- The main page at the search bar
#### How should this be manually tested?
- Go to the search bar, type in a word, and press the search button and the recipe cards that show up should have the search value in their name. Try searching another word and another word, too.
#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Questions/Comments:
- I want to add the ability to hit enter to search
